src/finch/_client.py

```diff
@@ -216,6 +223,72 @@ def copy(
     def __del__(self) -> None:
         self.close()
 
+    def forward(
+        self,
+        *,
+        method: str,
+        route: str,
+        data: Optional[str] | NotGiven = NOT_GIVEN,
+        headers: Optional[object] | NotGiven = NOT_GIVEN,
+        params: Optional[object] | NotGiven = NOT_GIVEN,
+        # Use the following arguments if you need to pass additional parameters to the API that aren't available via kwargs.
+        # The extra values given here take precedence over values defined on the client or passed to this method.
+        extra_headers: Headers | None = None,
+        extra_query: Query | None = None,
+        extra_body: Body | None = None,
+        timeout: float | None | NotGiven = NOT_GIVEN,
+    ) -> ForwardResponse:
+        """The Forward API allows you to make direct requests to an employment system.
+
+        If
+        Finch’s unified API doesn’t have a data model that cleanly fits your needs, then
+        Forward allows you to push or pull data models directly against an integration’s
+        API.
```

Comment:
Spacing is off here

Reply:
We removed these from now, but you'll need to make the changes you want for documentation like this in your OpenAPI spec before the next update for them to persist. Is that OK?